python_tests/pytest_input_generators.py

```diff
@@ -12,6 +12,138 @@
 from pytest_core import OpInfo, SampleInput, ErrorSample
 from nvfuser import DataType
 
+
+def broadcast_error_generator(
+    op: OpInfo, dtype: torch.dtype, requires_grad: bool = False, **kwargs
+):
+    # jax.lax.broadcast(operand, sizes)
+    # add new dimensions to left-hand-side of tensor
+    # dims = tuple(range(len(sizes), len(sizes) + np.ndim(operand)))
+    # return broadcast_in_dim(operand, tuple(sizes) + np.shape(operand), dims)
+
+    make_arg = partial(
+        make_tensor, device="cuda", dtype=dtype, requires_grad=requires_grad
+    )
+
+    fewer_original_axes = (
+        ([2, 3], [True, False]),
+        RuntimeError,
+        "Invalid broadcast, number of false entries in is_broadcast_dim expected to be",
+    )
+
+    greater_original_axes = (
+        ([2, 3], [True, False, False, False]),
+        RuntimeError,
+        "Invalid broadcast, number of false entries in is_broadcast_dim expected to be",
+    )
+
+    error_cases = [
+        fewer_original_axes,
```

Comment:
nitpick: IMHO, it's harder to read the error cases this way.

```
# exception inputs, exception type, exception string
error_cases = [
    (([2, 3], [True, False]), RuntimeError, "str"),
    (...),
]
```

Reply:
It is easier to disable test cases like this.